tidb-cloud/airbyte.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Connect Airbyte with TiDB
+summary: Learn how to use Airbyte TiDB connector.
+---
+
+# Connect Airbyte with TiDB
+
+[Airbyte](https://airbyte.com/) is an open-source data integration engine that helps you consolidate your data in your data warehouses, lakes, and databases. This document shows how to connect Airbyte to TiDB as a source or a destination.
+
+## Deploy Airbyte
+
+Airbyte supports local deployment with only a few steps.
+
+1. Install [Docker](https://www.docker.com/products/docker-desktop) on your workspace.
+
+2. Clone the Airbyte source code.
+
+    ```shell
+    $ git clone https://github.com/airbytehq/airbyte.git
+    $ cd airbyte
+    ```
+
+3. Run the Docker images by docker-compose.
+
+    ```shell
+    $ docker-compose up
+    ```
+
+Once you see an Airbyte banner, you can go to [http://localhost:8000](http://localhost:8000) to visit the UI.
+
+![img](/media/tidb-cloud/integration-airbyte-ready.jpg)
+
+## Set the TiDB connector
+
+If you use TiDB as the source, see a step-by-step tutorial about [Using Airbyte to Migrate Data from TiDB Cloud to Snowflake](https://www.pingcap.com/blog/using-airbyte-to-migrate-data-from-tidb-cloud-to-snowflake/).
+
+Conveniently, the steps are the same for setting TiDB as the source and the destination.
+
+1. Click **Sources** or **Destinations** in the sidebar and choose TiDB type to create a new TiDB connector.
+
+2. Fill in the parameters, such as host, port, database, username, and password.
+
+    - Host: The host domain of TiDB
+    - Port: The port of the database
+    - Database: The database that you want to sync the data
+    - Username: The username to access the database
+    - Password: The password of the username
+
+3. Enable **SSL Connection**, and set TLS protocols to **TLSv1.2** or **TLSv1.3** in **JDBC URL Params**.
+
+    > Note:
+    >
+    > - TiDB Cloud supports TLS connection. You can choose your TLS protocols in **TLSv1.2** and **TLSv1.3**, for example, `enabledTLSProtocols=TLSv1.2`.
+    > - If you want to disable TLS connection to TiDB Cloud via JDBC, you need to set useSSL to `false` in JDBC URL Params specifically and close SSL connection, for example, `useSSL=false`.
+    > - TiDB Serverless only supports TLS connections.
+
+4. Click **Set up source** or **destinations** to complete creating the connector.
+
+![img](/media/tidb-cloud/integration-airbyte-parameters.jpg)
+
+For more details about the TiDB connector, see [TiDB Source](https://docs.airbyte.com/integrations/sources/tidb) and [TiDB Destination](https://docs.airbyte.com/integrations/destinations/tidb).
+
```

Comment:
Do we need another section to tell users to set the destination ( or source, depending on which one you set up in the previouse section)?